pkg/http/http.go

```diff
@@ -163,7 +164,7 @@ func (c FileSDConfig) convert() (file.SDConfig, error) {
 }
 
 type AddressProvider interface {
-	Resolve(context.Context, []string)
+	Resolve(context.Context, []string) tsdberrors.MultiError
```

Comment:
You should never return concrete type of error in Go.

```suggestion
	Resolve(context.Context, []string) errors
```

Reply:
Sure, reverted it.